[logs] Note that archives don't support ChinaCloud (#15801)
DataDog/documentation: content/en/logs/log_configuration/archives.md

Set up the [Azure integration][1] within the subscription that holds your new storage account, if you haven't already. This involves [creating an app registration that Datadog can use][2] to integrate with.

> **Note:** We do not currently support archiving to Azure ChinaCloud.

Reviewer: ```suggestion
**Note:** Archiving to Azure ChinaCloud is not supported.
```
